### ISSUE TITLE
Trip report: mention sound system (1 hand) when thinking about live coding

### ISSUE THREAD

**Issue by thomas-keller** [TampaUseRs/speaker-starter-kit#5]
Despite going to several of these meetups, I didn't think about the sound system and not having two hands until I was actually on stage. That was just me being dumb, but I think it might be useful to mention that.

It might be worth considering putting some of the Rstudio money that would go into beer funds into a lavalier mic (one of those lapel mics) to make it a handless operation. They aren't too expensive: 

https://www.amazon.com/20-Channel-Wireless-Microphone-Transmitter-Performance/dp/B06XXW1GTZ/ref=sr_1_10?ie=UTF8&qid=1532534450&sr=8-10&keywords=wireless+lavalier+mic+system

**Comment by tgerke** [comment]
It's a great point and I totally agree: I actually shopped lavalier mics when I purchased the speaker last year, but didn't have enough funds to cover it at that time. Will make it happen if/when R Consortium gets back to me for this year's funding. I also learned last night that Southern has a mic system inside as well (I thought they only had outside), so I'll find out whether we can use that for upcoming sessions (and if it has a lavalier mic already).
